test/lib/test-ringpop-cluster.js

```diff
@@ -127,12 +126,66 @@ function testRingpopCluster(opts, name, test) {
     }
 
     tape(name, function onTest(assert) {
-        var cluster = bootstrapClusterOf(opts, function onBootstrap(results) {
-            assert.on('end', function onEnd() {
-                destroyCluster(cluster);
+        var tmpMinProtocolPeriods = [];
+        var cluster = createClusterOf(opts);
+        var joinResults;
+
+        // store protocol period. Set protocol period to 1 so that convergence is fast
+        cluster.forEach(function each(ringpop, i) {
```

Comment:
instead of doing this, wouldnt it be easier if in createClusterOf you had:

```
createRingpop(_.extend({
    host: '127.0.0.1',
    port: 10000 + i,
    minProtocolPeriod: 1
}, opts)));
```